proto/decentraland/sdk/components/audio_source.proto

```diff
@@ -13,11 +13,13 @@ option (common.ecs_component_id) = 1020;
 // Note that the `audio_clip_url` is not actually a URL, but rather the path of a file bundled with
 // the scene and declared in its manifest. The name was chosen because the URL use-case will
 // eventually be supported.
-message PBAudioSource { 
+message PBAudioSource {
   optional bool playing = 1; // whether the clip is currently playing.
   optional float volume = 2; // the audio volume (default: 1.0).
   optional bool loop = 3;    // whether the clip should restart when finished.
   optional float pitch = 4;  // the audio pitch (default: 1.0).
   string audio_clip_url = 5; // the clip path as given in the `files` array of the scene's manifest.
+  optional float current_time = 6;  // specifies the current playback time of the clip in seconds (default: 0).
+  optional float speed = 7;  // the playback speed (default: 1.0).
```

Comment:
what would this property do? and how it differs from `pitch`, can they be combined?

Reply:
actually yes

Reply:
removed